web-components/package.json

```
@@ -12,42 +12,36 @@
   },
   "exports": {
     ".": {
-      "import": "./dist/off-webcomponents.bundled.js",
-      "require": "./dist/off-webcomponents.bundled.cjs"
+      "import": "./dist/off-webcomponents.js",
+      "require": "./dist/off-webcomponents.cjs"
     }
   },
   "types": "dist/types/off-webcomponents.d.ts",
   "files": [
     "dist"
   ],
   "scripts": {
-    "build": "MODE=prod npm run translations:build && MODE=prod rollup -c",
-    "build:watch": "npm run translations:build && MODE=dev rollup -c -w",
-    "build:doc": "rm -rf docs/dist && npm run analyze && npm run build && cp -r dist docs && cp -f docs/index.html docs/dist/index.html",
+    "build": "MODE=prod npm run translations:build && vite build",
```

FYI, in this forç, MODE=prod is only available for the first command.

Maybe you should write:
```suggestion
    "build": "MODE=prod; npm run translations:build && vite build",
```